subprojects/core-api/src/main/java/org/gradle/api/provider/HasMultipleValues.java

```diff
@@ -97,7 +97,7 @@
      * @param elements The elements to add
      * @since 4.10
      */
-    @SuppressWarnings("varargs")
+    @SuppressWarnings("unchecked")
```

Comment:
What is the problem here? Should it be `@SafeVarargs` as present on the subtype?